tests/unit/test_my_annotator.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 
 from tests.unit import validate_threaded_result
+from vrs_anvil.translator import VCFItem
```

Comment:
should we remove the `my` to make this file `test_annotator.py`?

Reply:
done